home/templatetags/home_tags.py

```diff
@@ -55,6 +56,15 @@ def render_questionnaire_list(questionnaire):
 def render_user_progress(user_progress):
     return user_progress
 
+
 @register.inclusion_tag('home/tags/sub_sections.html')
 def render_sub_sections_list(sub_sections):
-    return {'sub_sections': sub_sections}
\ No newline at end of file
+    return {'sub_sections': sub_sections}
+
+
+@register.simple_tag
+def locale_set(locale, url):
+    for item in LANGUAGES:
+        code = item[0]
+        url = url.replace(f"/{code}/", "")
```

Comment:
I don't understand what this is doing. Does overwriting the value of the url in the loop help us somehow?